### PR TITLE
[Improvement][MOD-159] Add DOI message - Preprints

### PR DESCRIPTION
**Pull request by sheriefvt**
## Problem
Users don't know when they will have a DOI for their preprint.

## Solution
Add "DOI created after moderator approval." on pending pre-moderation preprints

image
image

## Ticket
https://openscience.atlassian.net/browse/MOD-159

### CODE REVIEW

**Comment by sheriefvt** [comment]
@aaxelb done 🐼

File: app/templates/content/index.hbs [hunk 1]
```diff
@@ -97,12 +97,18 @@
                         </div>
                     {{/if}}
 
-                    {{#if model.doi}}
-                        <div class="p-t-xs">
-                            <h4 class="p-v-md f-w-md"><strong>{{t "content.article_doi"}}</strong></h4>
+                    <div class="p-t-xs">
+                        {{#if model.doi}}
```
Comment (jamescdavis) [inline app/templates/content/index.hbs]: model.doi is actually the user-provided DOI for a peer-reviewed publication (not the preprint).

File: app/templates/content/index.hbs [hunk 2]
```diff
@@ -97,12 +97,18 @@
...
+                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
                             <a href={{doiUrl}} onlick={{action "click" "link" "Content - DOI" doiUrl}}>{{model.doi}}</a>
-                        </div>
-                    {{/if}}
+                        {{else if (not node.public)}}
+                            {{t 'content.preprint_pending_doi'}}
+                        {{else if (and model.provider.reviewsWorkflow (not model.isPublished))}}
+                            <h4 class="p-v-md f-w-md"><strong>{{t "content.article_doi"}}</strong></h4>
+                            {{t 'content.preprint_pending_doi_moderation'}}
+                        {{/if}}
```
Comment (jamescdavis) [inline app/templates/content/index.hbs]: This logic belongs in the section above this one that deals with `model.links.preprint_doi`

File: app/templates/content/index.hbs (hunk 1 above)
Comment (aaxelb) [reply app/templates/content/index.hbs]: It might help to use `model.articleDoiUrl` here instead

**Pull request merged**